lib/operations/is_capped.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const Operation = require('./operation').Operation;
+const OptionsOperation = require('./options_operation');
+const handleCallback = require('../utils').handleCallback;
+
+class IsCappedOperation extends Operation {
+  constructor(collection, options) {
+    super(options);
+
+    this.collection = collection;
+  }
+
+  execute(callback) {
+    const coll = this.collection;
+    const options = this.options;
+
+    const optionsOperation = new OptionsOperation(coll, options);
```

Comment:
Hmm, I'm curious why we are using composition here instead of inheritance? Could this not be 

```
class IsCappedOperation extends OptionsOperation {
  execute(callback) {
    super.execute((err, document) => {
      if (err) return handleCallback(callback, err);
      handleCallback(callback, null, !!(document && document.capped));
    });
  }
}
```